frontend/src/app/commonComponents/Alert.tsx

```diff
@@ -63,18 +64,19 @@ const Alert = ({
     return "region";
   };
 
-  const id = `${getIdentifiedRole()}-${Date.now()}`;
+  const bodyId = uuidv4();
 
   return (
     <div
       className={classes}
-      id={id}
+      id={uuidv4()}
       role={getIdentifiedRole()}
-      aria-labelledby={`${id}-body`}
+      aria-label={title ? `Alert: ${title}` : undefined}
```

Comment:
I am checking the aria role [description/guidance](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role) and it looks like we don't really need to define any aria-label/aria-labelledby to the wrapping role element as the purpose of setting the role is to trigger the screen reader to read the contents of it. I vote to remove the aria-label/aria-labelledby props completely.